docs/mcp_user_guide.md

```diff
@@ -0,0 +1,178 @@
+# AWS Deadline Cloud MCP Server User Guide
+
+## Architecture Overview
+
+The AWS Deadline Cloud MCP Server uses FastMCP within the MCP Python SDK to provide a simplified, decorator-based approach to tool registration while maintaining full MCP protocol compliance.
+
+
+## Project Structure
+
+The MCP server code is organized into modules by functional area:
+
+```
+src/deadline/mcp/
+├── __init__.py              # Package initialization
+├── server.py                # Main server with FastMCP setup
+└── tools/                   # Tool modules organized by function
+    ├── __init__.py          # Package initialization
+    ├── auth.py              # Authentication tools
+    ├── farm.py              # Farm management tools
+    ├── queue.py             # Queue management tools
+    ├── job.py               # Job management tools
+    ├── fleet.py             # Fleet management tools
+    ├── attachment.py        # Job attachment and manifest tools
+    └── placeholder.py       # Placeholder tools not supported in the API
+```
+
+## Tool Organization
+
+Tools are organized by functional area:
+
+### Authentication Tools (`src/deadline/mcp/tools/auth.py`)
+- `auth_status()`: Check current authentication status
+- `login()`: Log in to AWS Deadline Cloud using monitor credentials (non-blocking)
+- `logout()`: Log out from AWS Deadline Cloud monitor credentials (with timeout handling)
+
+### Farm Management Tools (`src/deadline/mcp/tools/farm.py`)
+- `list_farms()`: List available farms
+- `get_farm(farm_id: str)`: Get detailed farm information
+
+### Queue Management Tools (`src/deadline/mcp/tools/queue.py`)
+- `list_queues(farm_id: str)`: List queues in a farm
+- `get_queue(farm_id: str, queue_id: str)`: Get detailed queue information
+
+### Job Management Tools (`src/deadline/mcp/tools/job.py`)
+- `list_jobs(farm_id: str, queue_id: str)`: List jobs in a queue
+- `get_job(farm_id: str, queue_id: str, job_id: str)`: Get detailed job information
+- `submit_job(farm_id: str, queue_id: str, bundle_path: str, job_name: str = None, priority: int = 50)`: Submit new job from bundle (non-blocking with background processing)
+- `check_job_submission(submission_id: str)`: Check status of background job submission
+- `cancel_job(farm_id: str, queue_id: str, job_id: str, mark_as: str = "CANCELED")`: Cancel running job
+- `download_job_output(farm_id: str, queue_id: str, job_id: str, destination_dir: str = None)`: Download job output files
```

Comment:
I think this interface needs some more thought. What does `destination_dir` do when the job has inputs and outputs spread across three different drive letters?